test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -556,19 +559,26 @@ class test_ucp_sockaddr : public ucp_test {
         ep_params.sockaddr.addrlen = m_test_addr.get_addr_size();
         ep_params.user_data        = &sender();
 
+        if (specify_src_addr == true) {
```

Comment:
minor: no need `== true` for boolean type

Reply:
fixed